README.md

```diff
@@ -1530,6 +1530,16 @@ end
 
 While Procs are convenient for single cases, consider using [Custom Validators](#custom-validators) in cases where a validation is used more than once.
 
+Notice that [allow_blank](#allow_blank) validator is still relevant while using `values`. Having defined params like:
```

Comment:
Nit, let's use "note" like everywhere else, and make this into a single sentence?

_Note that [allow_blank](#allow_blank) validator applies while using `:values`. In the following example the absence of `:allow_blank` does not prevent `:state` from receiving blank values because `:allow_blank` defaults to `true`._

Reply:
Don't hesitate to fix the others ;)